test/update_all_fixtures.sh

```diff
@@ -11,7 +11,7 @@ for dir in examples/*/ ; do
     fi
 
     if [[ "$dir" == "examples/rules_ios/" ]]; then
-      bazel_version="6.4.0"
+      bazel_version="7.1.1"
     else
       bazel_version="last_green"
     fi
```

Comment:
I would remove this block entirely now.

Reply:
We have issues in rules_ios on HEAD. Can fix those up and open another PR once they're fixed